### PR TITLE
WindowCovering: Zap definition updates / fixes

### LINKED ISSUE #18333: [TE9][TC-GC-1.1] WindowCovering example: GeneralCommissioning cluster out-of-sync
#### Problem

Located in the WindowCovering example might be present elsewhere

<what's wrong or missing, please include any applicable:
 * FeatureMap is preset at 6 and should be 0
 * SupportsConcurrentConnection not present
 * SetRegulatoryConfigResponse not present
 >

 #### Proposed Solution
Need a patch in ZAP for alignement with the latest spec

### LINKED ISSUE #19804: [pre-sve] TC-GC-1.1/2.1 supports-concurrent-connection missing in Window Cover app
#### Problem
supports-concurrent-connection is not implemented in Window cover sample app.

 #### Commit
SHA: https://github.com/project-chip/connectedhomeip/commit/1e3f12041013dd71a53d035dbf2f6d142fc06c4e
Platform: EFR32

### PR DESCRIPTION
**Pull request by jmeg-sfy**
#### Problem
What is being fixed?  Examples:
* Definition is not complete and it is out synch or not compliant for SVE
* Fix https://github.com/project-chip/connectedhomeip/issues/19804
* Fix https://github.com/project-chip/connectedhomeip/issues/18333

#### Change overview
What's in this PR
* Add missing global attributes on WindowCovering Endpoints (window app + all-cluster-minimal)

* Fix GeneralCommissioning (ep 0) on window-app
* Update Identify Clusters (ep1 2) on window-app
* Update Groups Clusters  (ep1 2) on window-app
* Update Scenes Clusters  (ep1 2) on window-app
* Update Descriptor Clusters  (ep0 1 2) on window-app
* Remove useless Junk definition from BaseRootDevice type from (ep 1 2) (Basic Cluster, Thread, Network)

#### Testing
How was this tested? (at least one bullet point required)
* Test on EFR32 + passing CI

### CODE REVIEW

File: examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter [hunk 1]
```diff
@@ -3585,12 +3605,32 @@ endpoint 1 {
   }
 
   server cluster WindowCovering {
-    ram      attribute type;
+    ram      attribute type default = 0x08;
+    ram      attribute physicalClosedLimitLift default = 0xFFFF;
```
Comment (bzbarsky-apple) [inline examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter:3609]: This does not look right.  The minimal app is supposed to disable all optional functionality and enable as few features as possible while still being spec-compliant, and these bits are optional.  Please make sure only the things that are mandatory (given the Lift feature is enabled) are enabled here.

File: examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap [hunk 2]
```diff
@@ -13602,19 +13602,51 @@
               "storageOption": "RAM",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "0x00",
+              "defaultValue": "0x08",
```
Comment (bzbarsky-apple) [inline examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap:13605]: This is not right either.  This should be type 0, which is the simplest type that only supports the Lift feature.

File: examples/window-app/common/window-app.matter [hunk 3]
```diff
@@ -1891,10 +1940,22 @@ endpoint 1 {
...
     callback attribute clientList;
     callback attribute partsList;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
     ram      attribute featureMap;
     callback attribute clusterRevision default = 1;
   }
 
+  server cluster Binding {
```
Comment (bzbarsky-apple) [inline examples/window-app/common/window-app.matter:1950]: Why is this here?  I don't see Binding involved in this device type at all.

File: examples/window-app/common/window-app.matter [hunk 4]
```diff
@@ -1937,10 +2026,22 @@ endpoint 2 {
...
```
Comment (bzbarsky-apple) [inline examples/window-app/common/window-app.matter:2036]: Again, why is Binding here?

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter (hunk 1 above)
Comment (jmeg-sfy) [reply examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter:3609]: Ah ok i wasn't aware of this , i just wanted to aligned it , i ll remove

File: examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap (hunk 2 above)
Comment (jmeg-sfy) [reply examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap:13605]: gotcha

File: examples/window-app/common/window-app.matter (hunk 3 above)
Comment (jmeg-sfy) [reply examples/window-app/common/window-app.matter:1950]: https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/WindowCovering.adoc

Tell me if i am wrong
but this device type is classified as Simple
I was a bit lost with the requirements for Simple device there
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/BaseDeviceType.adoc
Normally i don't need it since it is for `Simple and Client`
Can you just confirm that neither Binding Client or Server are needed @bzbarsky-apple ? thanks

Comment (bzbarsky-apple) [reply examples/window-app/common/window-app.matter:1950]: You need a Binding server for a device type that:
1. Is Simple
2. Includes a client cluster.

because then we need some way to tell that client cluster which server(s) to talk to.  I'd really like to understand what the confusing part here was so we can fix it in the spec to be clearer.

The Window Covering device does not have any client clusters, so does not need a Binding server.  And it definitely does not need a binding Client: that's for the device to tell other devices who to talk to, and Window Covering is not going to be doing that, right?

So no, you don't need either side of Binding here @jmeg-sfy

Comment (jmeg-sfy) [reply examples/window-app/common/window-app.matter:1950]: Thanks for clarification ! I ll update

**Review by woody-apple — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
@jmeg-sfy That said, why were the all-clusters-app changes undone?  Are they not needed anymore?

**Comment by jmeg-sfy** [comment]
> @jmeg-sfy That said, why were the all-clusters-app changes undone? Are they not needed anymore

**Comment by woody-apple** [comment]
Fast tracking given this has had enough time for review.

Note: @jmeg-sfy there are conflicts, sorry :(

**Review by jmartinez-silabs — APPROVED** [review]
